.circleci/config.yml

```
@@ -0,0 +1,28 @@
+
+orbs:
+  maven: circleci/maven@2.0.0
+
+workflows:
+  maven_test:
+    jobs:
+      - maven/test:
+          executor: jdk
+          context: OpenAI
+      - build:
+          context: SonarCloud
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)